controllers/articles-controller.js

```diff
@@ -41,3 +42,19 @@ exports.getComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const votes = req.body;
+
+ const promises = [updateArticle(votes , article_id), checkArticleExists(article_id)];
+  Promise.all(promises)
```

Comment:
This is nice work but is not necessary here. We used this approach for the comments as it was possible for either an empty array representing the missing article or an empty array representing the missing comments to be returned. Here, the only scenario where we get an empty array returned is when the article does not exist. How could we change this?